images/service-serving-cert-signer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# This is the integrated OpenShift OService Serving Cert Signer.  It signs serving certificates for use inside the platform.
+#
+# The standard name for this image is openshift/origin-service-serving-cert-signer
+#
+FROM openshift/origin-base
+
+RUN INSTALL_PKGS="origin-service-serving-cert-signer" && \
```

Comment:
Remove `origin` or use `openshift` for the package name.  I guess we need *some* prefix though...